resources/contact.js

```diff
@@ -0,0 +1,60 @@
+// https://youtu.be/eg4e-FObyJ8
+const submitBtn = document.querySelector(".formBtn")
+
+
+submitBtn.addEventListener('click', function(){
+  
+  const name = document.querySelector("#name").value
+  const subject = document.querySelector("#subject").value
+  const email = document.querySelector("#email").value
+  const phone = document.querySelector("#phone").value
+  const message = document.querySelector("#message").value
+
+  var errorMessage = "Oops! Please Check the fields."
+  
+  if (validate(name, email, phone, message)){
+    var contactData = {name:name, email:email, phone:phone, message:message}
+    const contactJSON = JSON.stringify(contactData)
+    apiCall(contactJSON)
+  }
+  else{
+    alert(errorMessage)
+  }
+  
+})
+
+
+
+function validate(name, email, phone, message){
```

Comment:
Add proper error messages to form fields instead of the overall error message. [](https://developer.mozilla.org/en-US/docs/Learn/Forms/Form_validation)

Reply:
Error messages will be showed above their corresponding fields.